### PR TITLE
Add styling to comment notifications

### PR DESCRIPTION
**Pull request by Will-Howard**
Previously there was no styling in these emails

Screenshot 2022-10-28 at 14 53 13
Screenshot 2022-10-28 at 14 57 15

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1203259910689820) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/server/emailComponents/EmailComment.tsx [hunk 1]
```diff
@@ -69,19 +96,12 @@ const EmailCommentsOnTagHeader = ({tagId, isSubforum}: {tagId: string, isSubforu
...
-      New discussion comments on <a href={tagGetUrl(tag)}>{tag.name}</a>
-    </div>
-  );
+  const props = isSubforum
+    ? { text: `New comments in the ${startCase(tag.name)} subforum`, href: tagGetSubforumUrl(tag, true) }
+    : { text: `New discussion comments on ${tag.name}`, href: tagGetDiscussionUrl(tag) };
+  return <HeadingLink {...props} classes={classes}/>
 }
 
-const EmailCommentsOnPostHeaderComponent = registerComponent("EmailCommentsOnPostHeader", EmailCommentsOnPostHeader);
```
Comment (Will-Howard) [inline packages/lesswrong/server/emailComponents/EmailComment.tsx:83]: This wasn't used outside this file

File: packages/lesswrong/server/emailComponents/EmailFormatDate.tsx [hunk 2]
```diff
@@ -13,7 +13,7 @@ const EmailFormatDate = ({date, format}: {
   if (format)
     return <span>{moment(new Date(date)).format(format)}</span>
   else
-    return <span>{moment(new Date(date)).format("LLL z")}</span>
+    return <span>{moment(new Date(date)).format("LLL z").trim()}</span>
```
Comment (Will-Howard) [inline packages/lesswrong/server/emailComponents/EmailFormatDate.tsx:16]: There was a trailing space in the case that there is no timezone info

File: packages/lesswrong/server/emailComponents/EmailComment.tsx [hunk 3]
```diff
@@ -7,46 +7,75 @@ import './EmailFormatDate';
...
+const EmailCommentBatch = ({comments, classes}:{comments: DbComment[], classes: ClassesType}) => {
+  const { EmailComment } = Components;
   const commentsOnPosts = filter(comments, comment => !!comment.postId)
   const commentsByPostId = groupBy(commentsOnPosts, (comment:DbComment)=>comment.postId);
   const commentsOnTags = filter(comments, comment => !!comment.tagId && comment.tagCommentType === "DISCUSSION")
   const commentsByTagId = groupBy(commentsOnTags, (comment:DbComment)=>comment.tagId);
   const commentsOnSubforums = filter(comments, comment => !!comment.tagId && comment.tagCommentType === "SUBFORUM")
   const commentsBySubforumTagId = groupBy(commentsOnSubforums, (comment:DbComment)=>comment.tagId);
   
+  const commentsListComponent = (comments: DbComment[]) => {
```
Comment (oetherington) [inline packages/lesswrong/server/emailComponents/EmailComment.tsx:37]: Nit: Maybe this should be defined outside of this component? (Not a big deal)

**Review by oetherington — APPROVED** [review]
👍

**Pull request merged**